docs/basic_upgrade.md

```diff
@@ -0,0 +1,62 @@
+# Upgrade Basics
+
+
+You can upgrade rke2 by using the installation script, or by manually installing the binary of the desired version.
+
+>**Note:** When upgrading, upgrade server nodes first one at a time, then any worker nodes.
+
+### Release Channels
+
+Upgrades performed via the installation script or using our [automated upgrades](automated_upgrade.md) feature can be tied to different release channels. The following channels are available:
+
+| Channel |   Description  |
+|---------------|---------|
+|      stable     | (Default) Stable is recommended for production environments. These releases have been through a period of community hardening. |
```

Comment:
We are not going to have a stable channel upon initial release. Can you remove this table and instead have:

Currently, the `latest` channel is the only available channel. Once we have more releases and need to distinguish between the most recent release and the most stable release, we will add a `stable` channel and set it as the default.